.env.example

```diff
@@ -8,5 +8,6 @@ NEXTAUTH_SECRET=RAMDON_GENERATED_BASE64
 
 CRED_MANAGER_ROUTE="http://localhost:8080"
 SOURCECRED_INSTANCE_PATH="/instance"
+SOURCECRED_CACHE_DB_PATH="/instance/cache/sourcecred/github"
```

Comment:
I guess this also needs to be added on the secrets then right? Double check bc I think it also needs to be added on the action itself / .ENV file

Reply:
yes